www/_posts/2022-07-12-cordova-android-release-11.0.0.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 11.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 11.0.0`! This is one of Cordova's supported platforms for building Android applications.
+
+* [cordova-android@11.0.0](https://www.npmjs.com/package/cordova-android)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@11.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add android@11.0.0
+```
+
+* **Android 12 SplashScreen API Integration**
```

Comment:
It might be worth writing a separate post walking through (in detail, with screenshots and examples) how to migrate from the old splashscreen plugin to the new built-in one.